tests/unittest/cloud_governance/aws/zombie_non_cluster/test_ebs_in_use.py

```diff
@@ -6,7 +6,7 @@
 from cloud_governance.common.clouds.aws.s3.s3_operations import S3Operations
 from cloud_governance.policy.aws.ebs_in_use import EbsInUse
 
-os.environ['AWS_DEFAULT_REGION'] = 'us-east-1'
+os.environ['AWS_DEFAULT_REGION'] = 'us-east-2'
```

Comment:
Need to add [test_environment_variables.py]( https://github.com/redhat-performance/benchmark-runner/blob/main/tests/integration/benchmark_runner/test_environment_variables.py) for integration tests

Reply:
Added.
for the unit tests added a setter method to set.